assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -34,6 +34,9 @@ function useUploader() {
     allowedMimeTypes: {
       image: allowedImageMimeTypes,
       video: allowedVideoMimeTypes,
+    } = {
+      image: [],
+      video: [],
```

Comment:
Were there issues without this when running the plugin in a WordPress context?

If it's a storybook thing, then the proposed fix in #157 might make sense.

Otherwise, perhaps the config provider could have some defaults? Is a bit more  DRY.

Reply:
@wassgha can you please give me a direction on it?

Reply:
Reverted in a commit. No longer needed.